README.md

```diff
@@ -24,7 +24,10 @@ source ./venv/bin/activate
 pip3 install -r requirements.txt 
 ```
 
-- Please download Firebase Admin SDK service json file from Firebase Console and put that in project's root directory. Rename file to firebase_cred.json
+- Please download Firebase Admin SDK service json file from Firebase Console, You can download the file by following below steps:
+    -  Go to `Project Settings/Service accounts`
```

Comment:
@raghavjoshi789  As these are steps following an order, it should be a numbered list rather than a bullet list.

Reply:
@Rahulm2310 Why did you asked me to remove list numbering?

Reply:
>@raghavjoshi789 Rectify the list numbering.

@raghavjoshi789 I just told to rectify the list numbering as all the points were labelled `1`. I don't know why you replaced it with bullet list.